packages/graphql-key-transformer/src/sync_vtl_remove/Query.syncComments.req.vtl

```diff
@@ -0,0 +1,164 @@
+@key(fields:["content","createdAt"])
+@key(name: "byUsername", fields: ["username", "createdAt"], queryField: "commentsByUsername")
+@key(name: "byeditor", fields: ["editor", "createdAt"], queryField: "commentsByeditors")  
+{
+  id: ID!
+  content: String!
+  username: String!
+  createdAt: String!
+  editor: String!
+  data1: String!
+  data2:String!
+}
+####### schema #########
+
+## [Start] Set map initialization for @key **
+#set( $index = "" )
+#set( $scan = true )
+#set( $filterMap = {} )
+#set( $QueryMap = {} )
+#set( $PkMap = {} )
+#set( $PkTypeMap = {} )
+#set( $SkTypeMap = {} )
+#set( $PkType = {} )
+#set( $SkType = {} )
+## [End] Set map initialization for @key **
+## [Start] Set query expression for @key **
+$util.qr($QueryMap.put('content+createdAt' , 'dbTable'))
+$util.qr($PkMap.put('content' , 'dbTable'))
+$util.qr($PkTypeMap.put('content' , 'S'))
+$util.qr($SkTypeMap.put('createdAt' , 'S'))
+## [End] Set query expression for @key **
+## [Start] Set query expression for @key **
+$util.qr($QueryMap.put('username+createdAt' , 'byUsername'))
+$util.qr($PkMap.put('username' , 'byUsername'))
+$util.qr($PkTypeMap.put('username' , 'S'))
+$util.qr($SkTypeMap.put('createdAt' , 'S'))
+## [End] Set query expression for @key **
+## [Start] Set query expression for @key **
+$util.qr($QueryMap.put('editor+createdAt' , 'byeditor'))
+$util.qr($PkMap.put('editor' , 'byeditor'))
+$util.qr($PkTypeMap.put('editor' , 'S'))
+$util.qr($SkTypeMap.put('createdAt' , 'S'))
+## [End] Set query expression for @key **
+## [Start] Set query expression for @key **
+#if( !$util.isNullOrEmpty($ctx.args.filter.and) )
+  #set( $json = $ctx.args.filter.and )
+  #foreach( $item in $json )
+    #set( $ind = $foreach.index )
+    #foreach( $entry in $item.entrySet() )
+      #if( $ind == 0 && !$util.isNullOrEmpty($entry.value.eq) && !$util.isNullOrEmpty($PkMap.get($entry.key)) )
+        #set( $pk = $entry.key )
+        #set( $scan = false )
+        $util.qr($ctx.args.put($pk,$entry.value.eq))
+        #set( $index = $PkMap.get($pk) )
+        #set( $PkType = $PkTypeMap.get($pk) )
+      #end
+      #if( $ind == 1 && !$util.isNullOrEmpty($pk) && !$util.isNullOrEmpty($QueryMap.get("${pk}+$entry.key")) )
+        #set( $sk = $entry.key )
+        $util.qr($ctx.args.put($sk,$entry.value))
+        #set( $index = $QueryMap.get("${pk}+$sk") )
+        #set( $SkType = $SkTypeMap.get($sk) )
+      #else
+        #if( $ind > 0 )
+          $util.qr($filterMap.put($entry.key,$entry.value))
+        #end
+      #end
+    #end
+  #end
+#else
+  #set( $filterMap = $ctx.args.filter )
+#end
+## [End] Set query expression for @key **
+## [Start] Set Primary Key initialization @key **
+#set( $modelQueryExpression = {} )
+#if( !$util.isNull($pk) )
+  #set( $modelQueryExpression.expression = "#pk = :pk" )
+  #set( $modelQueryExpression.expressionNames = {
+  "#pk": "$pk"
+} )
+  #set( $modelQueryExpression.expressionValues = {
+  ":pk": {
+      "$PkType": "$ctx.args.get($pk)"
```

Comment:
Do we only need the PkType to see the expression? If so could we use the `$util.dynamodb.toDynamoDBJson()` utility here?
https://docs.aws.amazon.com/appsync/latest/devguide/resolver-util-reference.html#dynamodb-helpers-in-util-dynamodb